runtime/environment.go

```diff
@@ -149,7 +149,7 @@ var _ stdlib.Hasher = &interpreterEnvironment{}
 var _ ArgumentDecoder = &interpreterEnvironment{}
 var _ common.MemoryGauge = &interpreterEnvironment{}
 
-func newInterpreterEnvironment(config Config) *interpreterEnvironment {
+func NewInterpreterEnvironment(config Config) *interpreterEnvironment {
```

Comment:
Needed this to initialize the account-related handlers (e.g: during tests).
We would need to revisit this again on how we can better re-use these. For now, I was mostly trying to get things working.